docs/user-guides/desktop/index.md

```diff
@@ -121,7 +127,27 @@ Before you can use Coder Desktop, you will need to sign in.
 
 ## Troubleshooting
 
-Do not install more than one copy of Coder Desktop. To avoid system VPN configuration conflicts, only one copy of `Coder Desktop.app` should exist on your Mac, and it must remain in `/Applications`.
+If you encounter an issue with Coder Desktop that is not listed here, file an issue in the GitHub repository for
+Coder Desktop for [macOS](https://github.com/coder/coder-desktop-macos/issues) or
+[Windows](https://github.com/coder/coder-desktop-windows/issues), in the
+[main Coder repository](https://github.com/coder/coder/issues), or consult the
+[community on Discord](https://coder.com/chat).
+
+### Known Issues
+
+#### macOS: Do not install more than one copy of Coder Desktop
+
+To avoid system VPN configuration conflicts, only one copy of `Coder Desktop.app` should exist on your Mac, and it must remain in `/Applications`.
+
+#### Coder Desktop can't connect through another VPN
+
+If the logged in Coder deployment requires a VPN to connect, Coder Connect can't establish communication through the VPN,
+and will time out.
+
+This is due to known issues with how [macOS](https://github.com/coder/coder-desktop-macos/issues/201) and
+[Windows](https://github.com/coder/coder-desktop-windows/issues/147)
+network extensions communicate with each other.
```

Comment:
Only macos has "network extensions". On windows it's some other issue with the VPN code that I haven't fully diagnosed yet.